File: PB - More Exercises Python/2.0 Conditional Statements - More Exercises/06. Pets.py
from math import ceil, floor

days, kg_food, dog_food_kg, cat_food_kg, turtle_food_kg = [float(input())  for _ in range(5)]
food = (days * dog_food_kg) + (days * cat_food_kg) + (days * turtle_food_kg) / 1000

if food <= kg_food:
    print(f'{floor(kg_food - food)} kilos of food left.')
else:
    print(f'{ceil(food - kg_food)} more kilos of food are needed.')


'''
5
10
2.1
0.8
321
'''

File: PB - More Exercises GO/2.0 Conditional Statements - More Exercises/06. Pets.go
package main

import "fmt"
import "math"

func main() {
	var days, kgFood, dogFoodKg, catFoodKg, turtleFoodKg float64
	fmt.Scan(&days, &kgFood, &dogFoodKg, &catFoodKg, &turtleFoodKg)
	food := (days * dogFoodKg) + (days * catFoodKg) + (days*turtleFoodKg)/1000
	if food <= kgFood {
		fmt.Printf("%v kilos of food left.", math.Floor(kgFood-food))
	} else {
		fmt.Printf("%v more kilos of food are needed.", math.Ceil(food-kgFood))

	}
}

/*
name   :06. Pets

input  :2 10 1 1 1200
output :3 kilos of food left.

input  :5 10 2.1 0.8 321
output :7 more kilos of food are needed.
*/
